src/components/Cluster/ClusterDetail/KeypairCreateModal/KeyPairCreateModal.tsx

```diff
@@ -59,7 +59,7 @@ const KeyPairCreateModal: React.FC<IKeyPairCreateModalProps> = (props) => {
   });
 
   const confirmAddKeyPair = async (
-    e: React.FormEvent<HTMLFormElement>
+    e: React.FormEvent<HTMLFormElement> | React.MouseEvent<HTMLElement>
```

Comment:
Unrelated, but triggered a compiler error after the changes in the `Button` component.